match/lib/match/spaceship_ensure.rb

```diff
@@ -97,5 +99,43 @@ def profile_exists(username: nil, uuid: nil, platform: nil)
         return nil
       end
     end
+
+    # @return the name of the provisioning profile type
+    # The kind of provisioning profile we're interested in
+    def profile_types(prov_type)
```

Comment:
Could we extract this method to `match/lib/match/module.rb`? It looks like it has the same implementation as in `match/lib/match/nuke.rb`

Reply:
Yeah, this would be great if we could abstract this out!